arkouda/sorting.py

```diff
@@ -126,7 +123,7 @@ def coargsort(arrays: Sequence[Union[Strings, pdarray, 'Categorical']], algorith
             atypes.append(a.objtype)
         elif isinstance(a, Strings):
             atypes.append(a.objtype)
-            anames.append('{}+{}'.format(a.entry.name, "legacy_placeholder"))
+            anames.append(a.entry.name)
```

Comment:
No change requested: Just thinking down the line, I wonder if adding a `getName()` function to `pdarray` and `Strings` would simplify some of the casting / conditional logic.  In theory it would start to collapse to `pda.getname()`